frontend/src/Components/CreateItemForm.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import {Button,Container,Grid,TextField} from '@mui/material';
+
+
+export default function CreateItemForm(props){
+    const [newToDoName,setNewToDoName] = useState('');
+    const handleChange = (event) =>{
+        setNewToDoName(event.target.value);
+    }
+
+    const handleKeyPress = (key, newTodoName)=>{
+        if(key === 'Enter'){
+            handleSubmit(newTodoName);
+        }
+    }
```

Comment:
pass the whole event to the function so that you can call `event.stopPropagation()` on it which will prevent it from propagating (continuing up) to parent elements

Reply:
@JWA111 Should I add `event.stopPropagation()` to other handlers like my `onClick` functions in other components too by default? Is there a good way to tell when I should include this or exclude this?